editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1205,25 +1221,53 @@ function createExpressionOtherJavaScript(
   sourceMap: RawSourceMap | null,
   elementsWithin: ElementsWithin,
   alreadyExistingUIDs: Set<string>,
-): WithParserMetadata<JSExpressionOtherJavaScript> {
+): WithParserMetadata<JSExpressionMapOrOtherJavascript> {
+  const isList = (() => {
```

Comment:
I know this is a draft, but so that you don't miss this one later - this is now just using `isNodeAMapExpression`

Reply:
Yes, fixed